go.mod

```diff
@@ -29,6 +28,7 @@ require (
 	cosmossdk.io/errors v1.0.0-beta.7
 	cosmossdk.io/math v1.0.0-beta.2
 	github.com/celestiaorg/rsmt2d v0.6.0
+	github.com/cosmos/cosmos-sdk v0.0.0-00010101000000-000000000000
```

Comment:
We may consider this PR if it strictly upgraded testify and didn't modify the github.com/cosmos/cosmos-sdk dependency.

Reply:
yeah agree, I think this happens sometimes when we use a different version than our fork for some reason

https://github.com/celestiaorg/cosmos-sdk/blob/482d8986c7842b93f40239a210fac8b3975db39a/go.mod#L50